VirtoCommerce.InventoryModule.Web/Scripts/blades/inventory-detail.tpl.html

```diff
@@ -72,7 +72,8 @@
                         </div>
                     </div>
                 </div>
+                <va-metaform blade="blade" registered-inputs="blade.metaFields" ng-init="setForm(detailForm)"></va-metaform>
```

Comment:
ng-init="setForm(detailForm)" looks redundant, isn't it?

Reply:
Strange that it fell into commit. :) Fixed.